doc/api/timers.md

```diff
@@ -292,7 +292,23 @@ returned Promises will be rejected with an `'AbortError'`.
 
 For `setImmediate()`:
 
-```js
+```mjs
+import { setImmediate as setImmediatePromise } from 'node:timers/promises';
+
+const ac = new AbortController();
+const signal = ac.signal;
+
+setImmediatePromise('foobar', { signal })
+  .then(console.log)
+  .catch((err) => {
+    if (err.name === 'AbortError')
+      console.error('The immediate was aborted');
+  });
```

Comment:
In ESM we would use TLA
```suggestion
try {
 console.log(await setImmediatePromise('foobar', { signal }));
} catch (err) {
    if (err.name === 'AbortError')
      console.error('The immediate was aborted');
}
```

Reply:
Ah nevermind, we want to `ac.abort()` to fire in parallel. Might make sense to add a comment
```suggestion
// We do not `await` the promise so `ac.abort()` is called concurrently.
setImmediatePromise('foobar', { signal })
  .then(console.log)
  .catch((err) => {
    if (err.name === 'AbortError')
      console.error('The immediate was aborted');
  });
```

Reply:
Perfect! I added your suggestion!